.github/scripts/publish-screenshots.sh

```
@@ -0,0 +1,119 @@
+    done
+fi
+
+# Build the comment body.
+BODY_FILE="$(mktemp)"
+{
+    echo "${COMMENT_MARKER}"
+    echo "## Screenshots"
+    echo ""
+    echo "_Updated for [\`${HEAD_SHA:0:7}\`](https://github.com/${REPO}/pull/${PR_NUMBER}/commits/${HEAD_SHA})._"
+    echo ""
+    for png in $(ls "${STAGE}" | sort); do
```

Shellcheck (0.11.0)

[info] Use find instead of ls to better handle non-alphanumeric filenames.

(SC2012)